.github/workflows/unittests-rust.yml

```diff
@@ -0,0 +1,24 @@
+name: Rust unit tests
+
+on:
+  push:
+    # TODO: remove branches
+    branches: ["*"]
+
+jobs:
+  test-rust:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+            profile: minimal
+            toolchain: stable
+            override: true
```

Comment:
I'm also fine with depending on the Github default - we just want to run the tests, so you know better than me

Reply:
If we were specifying a version, it would make much sense, since the system is whatever you get, depends on the runner, and it could be updated.
But if any *sufficiently updated* version is good enough (as it is currently being installed by the action), I believe the action is not adding much (apart from runtime :P)

Reply:
Sooner or later we will have to engage into the MSRV  business for eko as well, but I'm not sure we want to do this here ... shall I drop the step for the moment? (just to have a quick PR)

Reply:
> Sooner or later we will have to engage into the MSRV business

Fine, but it doesn't mean that tests have to be run with MSRV (the hard requirement is >= MSRV). And we should try to keep MSRV as low as possible (without exaggerating...).

Reply:
I know - I just wanted to say, let's keep the version discussion for later and do the easy thing here :upside_down_face:

Reply:
Then, w/o version constraints, I see no reason for not using the built-in.

(btw, it may apply to Python as well, but that would save a single line of the script, and that action is provided by GitHub, so I believe it should be cached, and it should change practically nothing)
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#python